en/06_Project_Governance/03_Maintainer_Guidelines.md

```diff
@@ -36,23 +40,113 @@ often working alongside Core Committers.
 
 CMS Squad members have write access to core repositories in order to work effectively with GitHub issues. They are expected to use those permissions with good judgement for merging pull requests.
 
+### Peer Reviewers
+
+[info]
+This is a new role, and is currently in a trial period. It is likely to change over time as we learn what works and what doesn't.
+[/info]
+
+This role is made up of community members who have demonstrated a willingness to improve Silverstripe CMS through their contributions to the open source supported modules. It empowers community members to have a more active role in ensuring pull requests get merged, and builds more momentum for community-created pull requests.
+
+While anyone in the community can review a pull request, only reviews from maintainers have weight of authority - and only maintainers can meaningfully approve changes. The Peer Reviewer role ensures members of the community have that authority.
+
+#### Responsibilities {#peer-reviewer-responsibilities}
+
+* Review pull requests, ensuring documented processes and best practices are being followed
+  * Refer to [how to review pull requests](/contributing/triage_resources/#how-to-review-pull-requests) and [contributing code](/contributing/code/)
+* Escalate pull requests to Core Committers if the PR is very complex and you don't feel confident reviewing it
+  * If another reviewer feels confident, they can offer to review it instead
+* Escalate pull requests to Core Committers if the contributor is rude/argumentative/hard to deal with
+  * If you feel confident doing so, politely request the contributor behave in accordance with our [code of conduct](./code_of_conduct), and close the pull request if they refuse to do so
+* After two maintainers have approved the pull request, merge it
+  * The person who merges the PR must also understand and approve the changes
```

Comment:
> Are we saying that we need a 3 person to merge the PR here? Could the person doing the merging be one of the two peer reviewer?

This is explained in the very next line: "The person who contributed the PR must not be included as one of the two approving reviewers"

In other words:

Three people must be involved in the pull request, which include:
1. The person who creates the PR
2. A first person to approve the PR
3. A second person to approve the PR and then merge it

If the CMS Squad and Core Committers are merging, then only two people are required:
1. The person who creates the PR
2. The CMS Squad member or Core Committer who approves and merges the PR

Just as the CMS Squad and Core Committers are discourages from self-merging, peer reviewers shouldn't self-approve and self-merge.

> Do we need to clarify what "understand and approve the changes" means here.

I don't think so? "understand" is hopefully self-explanatory. "Approve" means you click the "approve" button. That doesn't seem ambiguous but if you think it is, please provide a specific suggestion and explain why it makes it clearer.

> If 2 senior contributors did an in-depth review of a change and tested it, I wouldn't want to need a third person to replicate their work.
>
> I would expect that 3rd person would do something like:
> - validate that the first two peer reviewers did their job as expected
> - do passing analysis to make sure there's no substantial concerns that went under the radar, without doing a full review.

The third person doesn't need to replicate anything. But if the peer reviewers aren't confident about the change, they shouldn't merge it. If they want a third opinion, they should get a third opinion. We don't need to dictate what level of involvement that third opinionator should have - it will vary depending on the level of confidence of the first two reviewers, the complexity of the PR, and other factors.

Reply:
Done, but with different wording